components/library/goffice/Makefile

```diff
@@ -44,19 +44,22 @@ COMPONENT_PREP_ACTION =        ( cd $(@D)  && \
 CONFIGURE_OPTIONS+=	--sysconfdir=/etc
 CONFIGURE_OPTIONS+=	--libexecdir=$(CONFIGURE_LIBDIR.$(BITS))
 
+CFLAGS+= -std=gnu99
+LDFLAGS += -lm
+
 CONFIGURE_ENV+= PYTHON=$(PYTHON)
 
 COMPONENT_INSTALL_ENV+= GCONF_DISABLE_MAKEFILE_SCHEMA_INSTALL=1
 
 # common targets
-build:		$(BUILD_32_and_64)
+build:		$(BUILD_64)
 
-install:	$(INSTALL_32_and_64)
+install:	$(INSTALL_64)
```

Comment:
It's a library. It was delivered for 32 bits and I think we should continue to deliver it for both architectures, unless there are strong reasons to avoid delivering 32-bit version.

Reply:
I doubt there is any consumer outside gnumeric and gnucash... since 32-bit tests fail not sure it is worth spending time on it.